### PR TITLE
enable windows build for edgemicroctl

### PR DESCRIPTION
**Pull request by srinandan**

### CODE REVIEW

File: cli/lib/gateway.js [hunk 1]
```diff
@@ -337,3 +340,49 @@ function hasConfigChanged(oldConfig, newConfig) {
...
+        return false;
+    }
+    if (typeof o === 'object' && o instanceof Object && !(o instanceof Array)) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+function checkArray(a) {
```
Comment (relloller) [inline cli/lib/gateway.js]: checkArray can be replaced with native Array.isArray

File: cli/lib/gateway.js [hunk 2]
```diff
@@ -337,3 +340,49 @@ function hasConfigChanged(oldConfig, newConfig) {
...
+    }
+
+    return true;
+}
+
+function checkObject (o) {
+    if (o === null) {
+        return false;
+    }
+    if (typeof o === 'object' && o instanceof Object && !(o instanceof Array)) {
```
Comment (relloller) [inline cli/lib/gateway.js]: can reduce number of lines by omitting if else to `return (typeof o === 'object' && o instanceof Object && !(o instanceof Array))`

**Comment by srinandan** [comment]
PTAL

**Comment by relloller** [comment]
LGTM

**Pull request merged**